src/ZoneServer/Skills/Combat/SkillModifier.cs

```diff
@@ -63,6 +63,14 @@ public class SkillModifier
 		/// </remarks>
 		public int HitCount { get; set; } = 1;
 
+		/// <summary>
+		/// Gets or sets forced critical status.
+		/// </summary>
+		/// <remarks>
+		/// If this is true, the attack always deals a critical hit.
+		/// </remarks>
+		public bool ForcedCritical { get; set; }
```

Comment:
Why add forced critical but not forced block, in preparation for when we need to move the code and will need it?

Reply:
I actually don't think a skill can add forced block, since block is something the defender does, not the attacker.  Forced Block comes from buffs.

Reply:
I think you mentioned somewhere that you understood what I was going for with this suggestion after seeing the combat event interfaces, right?^^

Reply:
Yes, I actually added it to the modifier class already.